webextensions/api/storage.json

```
@@ -465,6 +1059,220 @@
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/clear",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "15"
+                },
+                "firefox": {
+                  "version_added": "53"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            }
+          },
+          "get": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "15"
+                },
+                "firefox": {
+                  "version_added": "53"
+                },
+                "firefox_android": "mirror",
+                "opera": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            },
+            "empty_key": {
+              "__compat": {
+                "description": "Supports empty key",
+                "support": {
+                  "chrome": {
+                    "version_added": "≤105"
+                  },
+                  "edge": {
+                    "version_added": "15"
+                  },
+                  "firefox": {
+                    "version_added": "53"
+                  },
+                  "firefox_android": "mirror",
+                  "opera": {
+                    "version_added": false
+                  },
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
+          },
+          "getBytesInUse": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/getBytesInUse",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "15"
+                },
+                "firefox": {
+                  "version_added": false,
+                  "impl_url": "https://bugzil.la/1385832"
```

```suggestion
                  "version_added": "78"
```

`storage.sync.getBytesInUse()` was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1637166

(desktop only)